website/content/api-docs/agent/check.mdx

```diff
@@ -472,22 +465,24 @@ The table below shows this endpoint's support for
 | ---------------- | ----------------- | ------------- | -------------------------- |
 | `NO`             | `none`            | `none`        | `node:write,service:write` |
 
-### Parameters
+### Path Parameters
+
+- `check_id` `(string: "")` - Specifies the unique ID of the check to use.
 
-- `check_id` `(string: "")` - Specifies the unique ID of the check to
-  use. This is specified as part of the URL.
+### Query Parameters
+
+- `ns` `(string: "")` <EnterpriseAlert inline /> - Specifies the namespace in which to update the check.
+  The namespace can also be specified [through other methods](#methods-to-specify-namespace).
+
+### JSON Request Body Schema
 
 - `Status` `(string: "")` - Specifies the status of the check. Valid values are
   `"passing"`, `"warning"`, and `"critical"`.
 
 - `Output` `(string: "")` - Specifies a human-readable message. This will be
   passed through to the check's `Output` field.
 
-- `ns` `(string: "")` <EnterpriseAlert inline /> - Specifies the namespace in
-  which to update the check. This value can be specified as the `ns` URL query
-  parameter or the `X-Consul-Namespace` header. If not provided by either,
-  the namespace will be inherited from the request's ACL token or will default
-  to the `default` namespace. Added in Consul 1.7.0.
+<!-- Are we missing a Namespace field here? -->
```

Comment:
I couldn't tell whether the `Namespace` field existed in the JSON Request Body from looking at the code. But the `ns` query parameter definitely exists. I left this as a comment so someone else could look into this in the future if needed.